worldengine/draw.py

```diff
@@ -477,3 +554,8 @@ def draw_ancientmap_on_file(world, filename, resize_factor=1,
                     draw_biome, draw_rivers, draw_mountains, draw_outer_land_border, 
                     verbose)
     img.complete()
+
+def draw_scatter_plot_on_file(world, filename):
+    img = ImagePixelSetter(512, 512, filename)
```

Comment:
Yeah. I love huge maps as well. but it just makes no sense to make the plot huge. I'm going to include an example plot below so you can see what it looks like (I will probably make some changes in the future such as labeling the axis, but for now this should suffice).